CMakeLists.txt

```diff
@@ -67,6 +67,8 @@ endif()
 option(LLAMA_ACCELERATE             "llama: enable Accelerate framework"                    ON)
 option(LLAMA_OPENBLAS               "llama: use OpenBLAS"                                   OFF)
 option(LLAMA_CUBLAS                 "llama: use cuBLAS"                                     OFF)
+set(LLAMA_CUDA_BY "1" CACHE STRING  "llama: y block size for dmmv CUDA kernels")
```

Comment:
Can avoid introduce more and more options? instead to check the compute capability to decide if we should enable these features dynamically.

Reply:
Automating performance optimizations is something that I would like to do long-term but right now I don't think we have the data necessary to judge which options should be enabled under which circumstances.

Reply:
I think there should be more options. There are plenty already for runtime tuning.